lib/web_ui/lib/src/engine/font_fallbacks.dart

```diff
@@ -39,11 +41,17 @@ class FontFallbackManager {
 
   final List<NotoFont> fallbackFonts;
 
-  final NotoFont _notoSansSC;
-  final NotoFont _notoSansTC;
-  final NotoFont _notoSansHK;
-  final NotoFont _notoSansJP;
-  final NotoFont _notoSansKR;
+  // By default, we use the system language to determine the user's preferred
+  // language. This can be overridden through [debugUserPreferredLanguage] for testing.
+  String _language = domWindow.navigator.language;
+
+  @visibleForTesting
+  String get debugUserPreferredLanguage => _language;
+
+  @visibleForTesting
+  set debugUserPreferredLanguage(String value) {
+    _language = value;
+  }
```

Comment:
```suggestion
  @visibleForTesting
  String debugUserPreferredLanguage = _language;
```

Same effect, less code? :)

Reply:
Not really. With your suggestion, `debugUserPreferredLanguage` will get out of sync with `_language`, and the code below won't pick up changes to `debugUserPreferredLanguage`.